### PR TITLE
All files of javascript-2 week-2 homework

### PR DESCRIPTION
**Pull request by sayali05**

### CODE REVIEW

File: javascript/javascript2/week2/index.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+				<div class="total">
+					<p>Total price:<span id="totalPrice"></span></p>
+				</div>
+				<ul></ul>
+			</section> -->
+		</main>
+		<script src="hyfBayHelpers.js"></script>
+		<script src="main.js"></script>
+	</body>
+</html>
```
Comment (dinythomas89) [inline javascript/javascript2/week2/index.html:84]: Can you use a formatter like prettier?

File: javascript/javascript2/week2/main.css [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+}
+
+section.products > ul > li > ul {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	padding: 12px 12px 12px 12px;
+}
+
+section.products > ul > li > ul > li:first-child {
```
Comment (dinythomas89) [inline javascript/javascript2/week2/main.css:98]: Too much of nesting

File: javascript/javascript2/week2/main.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+	renderFilterProducts();
+});
+
+maxPriceTag.addEventListener("input", () => {
+	renderFilterProducts();
+});
+
+sortOption.addEventListener("change", () => {
+	renderFilterProducts();
+});
```
Comment (dinythomas89) [inline javascript/javascript2/week2/main.js]: These can be written as `htmlElement.addEventListener("change",renderFilterProduct)`

**Review by dinythomas89 — APPROVED** [review]

File: javascript/javascript2/week2/index.html (hunk 1 above)
Comment (sayali05) [reply javascript/javascript2/week2/index.html:84]: Yes, i have used prettier formatter.

File: javascript/javascript2/week2/main.js (hunk 3 above)
Comment (sayali05) [reply javascript/javascript2/week2/main.js]: Thank you so much for guidence. I have change code as per your suggestion.

**Pull request merged**